frontend/src/pages/modelRegistrySettings/ModelRegistrySettings.tsx

```diff
@@ -57,14 +65,17 @@ const ModelRegistrySettings: React.FC = () => {
       >
         <ModelRegistriesTable
           modelRegistries={modelRegistries}
-          refresh={refresh}
-          onCreateModelRegistryClick={() => setCreateModalOpen(true)}
+          refresh={refreshAll}
+          onCreateModelRegistryClick={() => {
+            setCreateModalOpen(true);
+            return Promise.resolve();
```

Comment:
Why do you need this `return Promise.resolve();`?

Reply:
Removed